sys/include/psa_crypto/psa/crypto_sizes.h

```diff
@@ -456,7 +456,30 @@ extern "C" {
  *
  *          See also @ref PSA_MAC_LENGTH().
  */
-#define PSA_MAC_MAX_SIZE (PSA_HASH_MAX_SIZE)
+#if (IS_USED(MODULE_PSA_MAC_HMAC_SHA_512) || \
+     IS_USED(MODULE_PSA_MAC_HMAC_SHA3_512))
+#define PSA_MAC_MAX_SIZE (PSA_HASH_LENGTH(PSA_ALG_SHA3_512))    /* 64 */
```

Comment:
Just for my understanding: The former definition was an over-approximation and the new one(s) narrows it down to the actual maximal MAC length according to the selected MAC modules, right?

Reply:
Yes

Reply:
Okay, thanks for the confirmation :)